pkg/apiclient/apiclient.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/base64"
 	"errors"
 	"fmt"
+	"google.golang.org/grpc/metadata"
```

Comment:
Please use another import group for this import (have a look in the CI build logs for why it failed)

Reply:
I'm sorry, I couldn't not find any info regarding that specific error, I even ran `make pre-commit` and couldn't find anything. Be that as it may, I guess the "metadata" import is used somewhere else within the same package? Therefore I used the "md" import group instead to reference this dependency, let me know if that's ok.

Reply:
The linter "silently" fixes small formatting issues and the order of the imports in the source files. A good practice therefore is, to run `pre-commit` (as the name implies) before commiting your changes. Or to run it after your (local) commit, and then commit the changes made by the linter (or rather, the `gofmt` sub-linter) before you push to your upstream repository if you want to see the changes made by `gofmt`.

Renaming the import namespace to `md` is not required and shouldn't be done in this case neither, imho. The error in the CI is all about where the import statement actually sits in the list of imports, because the `build` workflow runs the linter on your branch and errors out if there is a difference between your code and the code after the linter has finished.

Hope that clarified it a bit :)

Reply:
Thanks for the explanation, I just reordered the list of imports within apiclient.go!